backend/tutoring/views.py

```diff
@@ -134,12 +135,18 @@ def schedule(request):
                 'department_classes': department_classes,
             },
         )
-    except templatedoesnotexist:
+    except:
```

Comment:
This is not clean code.  I've personally spent ~2 hours debugging code and frustrated as shit because of a generic catch.  Read more here:
https://www.google.com/search?q=why+catch+all+excepts+are+bad&oq=why+catch+all+excepts+are+bad&aqs=chrome..69i57.4655j0j7&sourceid=chrome&ie=UTF-8

Reply:
Was the reason behind the modification the fact that 'templatedoesnotexist' is not a real error name, or that other error types need to be handled? If the former, we could presumably just correct the typo to TemplateDoesNotExist.

Reply:
Feel free to investigate and add a commit to this PR